controllers/events.js

```diff
@@ -193,11 +195,77 @@ const endActiveEvent = async (req, res) => {
   }
 };
 
+/**
+ * Adds a peer to an event.
+ *
+ * @async
+ * @function
+ * @param {Object} req - The Express request object.
+ * @param {Object} res - The Express response object.
+ * @returns {Promise<Object>} The JSON response with the added peer data and a success message.
+ * @throws {Object} The JSON response with an error message if an error occurred while adding the peer.
+ */
+const addPeerToEvent = async (req, res) => {
+  try {
+    const data = await eventQuery.addPeerToEvent({
+      peerId: req.body.peerId,
+      name: req.body.name,
+      role: req.body.role,
+      joinedAt: req.body.joinedAt,
+      eventId: req.params.id,
+    });
+    return res.status(200).json({
+      data,
+      message: `Selected Participant is added to the event.`,
+    });
+  } catch (error) {
+    logger.error({ error });
+    return res.status(500).json({
+      error: error.code,
+      message: "You can't add selected Participant. Please ask Admin or Host for help.",
+    });
+  }
+};
+
+/**
+ * Kicks out a peer from an event.
+ *
+ * @async
+ * @function
+ * @param {Object} req - The Express request object.
+ * @param {Object} res - The Express response object.
+ * @returns {Promise<Object>} The JSON response with a success message if the peer is successfully kicked out.
+ * @throws {Object} The JSON response with an error message if an error occurred while kicking out the peer.
+ */
+const kickoutPeer = async (req, res) => {
+  const { id } = req.params;
+  const payload = {
+    peer_id: req.body.peerId,
+    reason: req.body.reason,
+  };
+
+  try {
+    await apiService.post(`/active-rooms/${id}/remove-peers`, payload);
```

Comment:
taking the routes out in constant would become easier to maintain

Reply:
Thank you. It is fixed in this PR: https://github.com/Real-Dev-Squad/website-backend/pull/1387